### DISCUSSION TITLE
I facing these error while run the npx run hardhat

### DISCUSSION THREAD

**Discussion by Shaiks123** [NomicFoundation/hardhat#2862] [Troubleshooting] [answered]
npm WARN config global `--global`, `--local` are deprecated. Use `--location=global` instead.
Watching C:\Users\DELL\Desktop\PracticeRolex and all sub-directories not excluded by your .gitignore. Will not monitor dotfiles.
Found & ignored ./node_modules ; is listed in .gitignore

Starting: hardhat
node:internal/modules/cjs/loader:936
  throw err;
  ^

Error: Cannot find module 'C:\Users\DELL\Desktop\PracticeRolex\hardhat'
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
    at Function.Module._load (node:internal/modules/cjs/loader:778:27)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:77:12)
    at node:internal/main/run_main_module:17:47 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}

**Comment by fvictorio** [comment, marked as answer]
You have to do `npx hardhat`, not `npx run hardhat`. Did you copy that command from somewhere? Maybe there's an error in our docs or in some community resource.
